src/SourceBuild/tarball/content/README.md

```diff
@@ -0,0 +1,152 @@
+# dotnet/dotnet - Home of the .NET VMR (experimental)
```

Comment:
Have we decided (after the experiment concludes) if we're going to have one VMR per version? It could be useful to communicate our plans on this front.

Reply:
Fair point. We landed on "we're not going to name this `dotnet/dotnet8` and we'll see how things look later in the cycle to see if we need to split" so not sure we have a clear strategy to share?

Reply:
I think a statement on "these are the two options we considered" could be useful. Versioning is a pretty relevant topic to any user.

Reply:
This is a good point. Poeple might just not know how new stuff is inside.

Should we maybe then declare that we expect that the VMR will host a full Linux build of .NET 8? So that people understand what version of .NET and when they can get out of it?

Should we explain that you can find the latest main in the main branch (and explain servicing branches)

Reply:
Also dealt with in the other thread